### PR TITLE
Incorporate set of templates in SDK

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: models/schema.go [hunk 1]
```diff
@@ -9,28 +9,75 @@ type Schema struct {
...
+			"bds":           []interface{}{},
+			"filters":       []interface{}{},
+			"externalEpgs":  []interface{}{},
+			"serviceGraphs": []interface{}{},
+		}
+		result = []map[string]interface{}{
+			templateMap,
+		}
+	} else {
+		templateMap := make(map[string]interface{})
```
Comment (lhercot) [inline models/schema.go:37]: Do you need this line?

File: models/schema.go [hunk 2]
```diff
@@ -9,28 +9,75 @@ type Schema struct {
...
+		for _, map_values := range template {
+			map_template := make(map[string]interface{})
```
Comment (lhercot) [inline models/schema.go:39]: Could this be initialized as the list in 45 so you don't need to merge it later?

**Review by lhercot — CHANGES_REQUESTED** [review]

File: models/schema.go (hunk 1 above)
Comment (shrsr) [reply models/schema.go]: Removed it

File: models/schema.go (hunk 2 above)
Comment (shrsr) [reply models/schema.go]: Yes..Done

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**